Users/usersRouter.js

```diff
@@ -0,0 +1,110 @@
+const router = require('express').Router();
+//Connect to usersmodel
+const DB = require('./usersModel');
+
+//Importing the restricted middleware to verify token
+const verify = require('../auth/restricted-middleware');
+
+//To run the server locally install npm and express
+
+//==========================================================================
+//Login user or create a new user if a current user is not found
+router.post('/', verify, async (req, res) => {
+  // console.log('Passed middleware from HTTP request: ', req.body.decodedToken);
+  //Deconstructing decoded token
+  const userInfo = req.body.decodedToken;
+  const name = userInfo.name;
+  const email = userInfo.email;
+  const uid = userInfo.uid;
+
+  //TODO:
+  //
+  // [x] Pass the rest of user info life first and last name
+  // [x] FIX DATE TO USE JS FUNCTION
+  // [x] Handle errors correctly
+  // [x] Hide private keys from firebase
+
+  //Date format for timestamp
+  var timeStamp = new Date();
+  var dd = timeStamp.getDate();
+  var mm = timeStamp.getMonth() + 1;
+  var yyyy = timeStamp.getFullYear();
+  today = yyyy + '-' + mm + '-' + dd;
```

Comment:
does this need to say `var today = `?

Reply:
Nice catch, it needs to be :
`timeStamp = yyyy + '-' + mm + '-' + dd;`